src/dawn/Optimizer/PassInlining.cpp

```diff
@@ -465,6 +466,8 @@ bool PassInlining::run(const std::shared_ptr<StencilInstantiation>& stencilInsta
   // Nothing to do ...
   if(strategy_ == IK_None)
     return true;
+  if(stencilInstantiation->getOptimizerContext()->getOptions().Debug)
```

Comment:
this is very hidden here. 
Can we move it to where we push all the passes?  If in debug we should not call the pass, isnt?